README.md

```diff
@@ -543,6 +575,12 @@ request. After setting errors for an pb_utils.InferenceResponse
 object, use InferenceResponseSender.send() to send response with the
 error back to the user.
 
+Starting from 23.10, request cancellation can be checked directly on the
+`InferenceResponseSender` object, for example `response_sender.is_cancelled()`,
```

Comment:
`InferenceResponseSender` object using `response_sender.is_cancelled()`.
If `response_sender.is_cancelled()` returned `True`, then no further steps
are needed to be performed on this object.

Reply:
Updated: [Documentation wording updates](https://github.com/triton-inference-server/python_backend/pull/304/commits/3a43ef3a9d0550935ae9857f3c3d54e73987ab35)